src/moderatorFrontendMain/kotlin/views/locations/locationsOverview/ListLocations.kt

```diff
@@ -136,7 +136,20 @@ class ListLocations : RComponent<ListLocationsProps, ListLocationsState>() {
           attrs.color = "primary"
           attrs.onClick = {
             setState {
-              window.open("/location/list/qr-codes", target = "_blank")
+              window.open("/location/qr-codes/checkout", target = "_blank")
```

Comment:
pls use muiButton.attrs.href (+ attrs.target) in case we do just links

Reply:
Just copied the other buttons 😅 Will do